### DISCUSSION TITLE
Setting to hide all/show all sections?

### DISCUSSION THREAD

**Discussion by Marxsal** [kookma/TW-Section#28] [General]
When you open up a sectioned tiddler, it always shows all the sections completely opened. Is there any way to close them all at once?

Thanks!

**Comment by kookma** [comment]
I am not sure who, but one of the use posted a small code to close all sections at once! Have a look at TW-Section announcement post on Talk.
If you look at the code, I used details widget. I have hard coded initial status as open!
You can change it! Or you can add a button to manually close all at once!
I thought this may clutter the UI, so I did add the button intentionally, but it seems for very long tiddler with many section this make sense!
